indico/htdocs/js/indico/modules/surveys.js

```diff
@@ -53,6 +54,21 @@
         $('.js-add-question-dropdown').dropdown();
     }
 
+    function setupSubmissionButtons() {
+        $('.js-delete-submission').on('indico:confirmed', function(e) {
+            var $this = $(this);
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                success: function(data) {
+                    location.reload(true);
```

Comment:
that's is quite ugly, especially with the `nocache=true` argument. Why not simply remove the submission row?